### PR TITLE
Collect all artifacts created and upload to GCP with release-manager

### PR DESCRIPTION
**Pull request by andsel**
## Release notes

[rn:skip]

## What does this PR do?

Downloads all artifacts generated for ARM and x86 by `ci/dra_x86_64.sh` `ci/dra_aarch64.sh`, position in locations expected by the `release-manager` and invokes it to upload to the global bucket.

## Why is it important/What is the impact to the user?

Doesn't impact the user.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- ~~[ ] I have made corresponding changes to the documentation~~
- ~~[ ] I have made corresponding change to the default configuration files (and/or docker env variables)~~
- ~~[ ] I have added tests that prove my fix is effective or that my feature works~~

## Author's Checklist

- [x] run on real CI

## Related issues

-

### CODE REVIEW

File: ci/dra_upload.sh [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+mv build/logstash-${STACK_VERSION}-docker-image-aarch64.tar.gz .
+mv build/logstash-oss-${STACK_VERSION}-docker-image-aarch64.tar.gz .
+mv build/logstash-ubi8-${STACK_VERSION}-docker-image-aarch64.tar.gz .
+
+mkdir -p build/distributions/dependencies-reports/
+mv build/logstash-${STACK_VERSION}.csv build/distributions/dependencies-${STACK_VERSION}.csv
+
+# set required permissions on artifacts and directory
+chmod -R a+r build/*
+chmod -R a+w build
```
Comment (kaisecheng) [inline ci/dra_upload.sh:85]: chmod $PWD has covered $PWD/build, are these redundant?

**Review by kaisecheng — APPROVED** [review]
left a comment on chmod, otherwise looks good to me. Thanks for your contribution to DRA 💟

File: ci/dra_upload.sh [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+mv build/logstash-oss-${STACK_VERSION}-docker-image-x86_64.tar.gz .
+mv build/logstash-ubi8-${STACK_VERSION}-docker-image-x86_64.tar.gz .
+mv build/logstash-${STACK_VERSION}-docker-image-aarch64.tar.gz .
+mv build/logstash-oss-${STACK_VERSION}-docker-image-aarch64.tar.gz .
+mv build/logstash-ubi8-${STACK_VERSION}-docker-image-aarch64.tar.gz .
+
+mkdir -p build/distributions/dependencies-reports/
+mv build/logstash-${STACK_VERSION}.csv build/distributions/dependencies-${STACK_VERSION}.csv
+
+# set required permissions on artifacts and directory
```
Comment (jsvd) [inline ci/dra_upload.sh:83]: can we explain why this is needed? are the files created only with user + group permissions and is some other user reading them?

Comment (andsel) [reply ci/dra_upload.sh:83]: the part
```
chmod -R a+r build/*
chmod -R a+w build
```
is an explicit request from DRA

File: ci/dra_upload.sh (hunk 1 above)
Comment (andsel) [reply ci/dra_upload.sh:85]: yes, was added later. Is due to the facts that the release-manager has to be able to write sha512 files for the files present in $PWD, specifically the `docker-image` stuff.

**Review by jsvd — APPROVED** [review]
LGTM! lovely to see the progress on this :)

**Pull request merged**